dpctl/enum_types.py

```diff
@@ -75,6 +75,19 @@ class backend_type(Enum):
 
 
 class event_status_type(Enum):
+    """
+    An enumeration of SYCL event states.
+
+    ==================   ============
```

Comment:
Is there a way to automate generating this table? Do users care about the value of an enum?

Reply:
My worry is next time we update the enum, the table is not going to be updated and will be incorrect. It has happened to backend and device type.

Reply:
I think it is fine. The `enum` values has to be added to Python class `event_status_type` anyway, and docstring should be updated while extending the enum.

Reply:
Ok. Can you fix the docs for the other two enums as well in that case 😄

Reply:
Done. Removed the table, used that space to give an example.